package.json

```diff
@@ -25,12 +25,14 @@
   "homepage": "https://github.com/finsweet/sweetjs#readme",
   "devDependencies": {
     "@types/highlightjs": "^10.1.0",
+    "@types/js-cookie": "^3.0.1",
     "esbuild": "^0.14.0",
     "prettier": "^2.2.1",
     "typescript": "^4.2.3"
   },
   "dependencies": {
     "@finsweet/webflow-addons": "^1.0.8",
-    "highlight.js": "^10.7.1"
+    "highlight.js": "^10.7.1",
+    "js-cookie": "^3.0.1"
```

Comment:
If the script is imported via CDN, you only need to install the @types, no need to install the package itself.

Reply:
got it 👌